src/shared_modules/keystore/include/keyStore.hpp

```diff
@@ -58,6 +58,10 @@ class TKeystore final
         }
 
         keystoreDB.put(key, rocksdb::Slice(encryptedValue), columnFamily);
+        logDebug2(KS_NAME,
```

Comment:
Let's remove this log.
1- Out of scope.
2- Terms like Encryption, column family are implementation details.
3- Printed in stdout of the tool.

Reply:
I wasn't sure to include that log either. Removed.